feat: log using custom msg (#31)
Eomm/mercurius-logging: README.md

```

### logMessage

If you want to put a custom message inside the log output, you can set this option as a `function(context)` which returns a `string` or `string array`. If function provided doesn't return an allowed value, log message will be `undefined`.

#### Example returning a string

```js
app.register(mercuriusLogging, {
  logMessage: function (context) {
    return `This is a request made with method ${context.reply.request.method}`
  }
})
```

Here's an output example

```json
{
  "level": 30,
  "graphql": {
    "queries": [
      "firstQuery:myTeam",
      "secondQuery:myTeam"
    ],

>     "msg": "This is a request made with method POST"
>   }

Reviewer: ```suggestion
  },
  "msg": "This is a request made with method POST"
```

}
```

#### Example returning an array

```js
app.register(mercuriusLogging, {
  logMessage: function (context) {

>     return [`This is a request made with method ${context.reply.request.method} by foo%s` , `bar` ]

Reviewer: ```suggestion
    return ['This is a request made with method %s by foo%s', context.reply.request.method, 'bar' ]
```

  }
})
```

Here's an output example

```json
{
  "level": 30,
  "graphql": {
    "queries": [
      "firstQuery:myTeam",
      "secondQuery:myTeam"
    ],

>     "msg": "This is a request made with method POST by foobar"
>   }

Reviewer: ```suggestion
  },
  "msg": "This is a request made with method POST by foobar"
```
